docker-compose.yml

```
@@ -0,0 +1,37 @@
+version: "3.8"
+
+services:
+  mysqldb:
+    platform: linux/x86_64
+    image: mysql
+    restart: no
+    env_file: ./.env
+    environment:
+      MYSQL_DATABASE: ${MYSQLDB_DATABASE}
+      MYSQL_USER: ${MYSQLDB_USER}
+      MYSQL_PASSWORD: ${MYSQLDB_PASSWORD}
+      MYSQL_ROOT_PASSWORD: ${MYSQLDB_ROOT_PASSWORD}
+    ports:
+      - ${MYSQLDB_LOCAL_PORT}:${MYSQLDB_DOCKER_PORT}
+    healthcheck:
+      test: ["CMD-SHELL", "mysqladmin ping -h localhost -P ${MYSQLDB_DOCKER_PORT} -u ${MYSQLDB_USER} -p ${MYSQLDB_ROOT_PASSWORD}"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+  app:
+    depends_on:
+      - mysqldb
+    restart: no
+    image: books-service
+    build: .
+    env_file: ./.env
+    ports:
+      - ${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}
+      - ${DEBUG_PORT}:${DEBUG_PORT}
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url": "jdbc:mysql://mysqldb:${MYSQLDB_DOCKER_PORT}/${MYSQLDB_DATABASE}?serverTimezone=UTC", 
+        "spring.datasource.username": "${MYSQLDB_USER}", 
+        "spring.datasource.password": "${MYSQLDB_PASSWORD}"
```

It is more question than suggestion, but in my case, it doesn't work without setting `jwt.secret`:
```suggestion
        "spring.datasource.password": "${MYSQLDB_PASSWORD}"
        "jwt.secret": "${JWT_SECRET}"
```

---

Mine worked without jwt.secret, but at one point I thought about adding it here.